### PR TITLE
feat(actions): add live-value View sub-modes to setup actions (#541)

### LINKED ISSUE #541: feat(actions): live-value View mode for setup adjustment actions (dc* telemetry readout)
### What would you like?

For each directional setup action, add a read-only **View …** sub-mode that renders the current `dc*` telemetry value on the key icon. View sub-modes live in the **same action** as the adjustment — switching the action's Setting dropdown to "View Brake Bias" turns that key into a read-only value readout for brake bias. Per-setting telemetry plumbing is done as each View sub-mode is implemented; no batch pre-wiring.

### Use case

Drivers who adjust in-car settings (brake bias, TC, ABS, fuel mixture, etc.) want a visible readout on the Stream Deck so they can confirm the current value without looking at the in-car display. [iRaceIT](https://iraceit.itxware.de/iraceit-the-streamdeck-plugin/incar/) does this; the request is for parity. Pairs nicely with #540 (a tap-only key adjusts, an adjacent View key shows the resulting value).

### Behaviour change

**Per setup action**

- Add "View …" entries to the existing Setting dropdown, **above** the Increase / Decrease entries (e.g. View Brake Bias / Increase Brake Bias / Decrease Brake Bias / View Brake Bias Fine / …).
- View modes are read-only: hide the Direction row, hide the "Enable dual-press mode" checkbox (from #540), and the action's `onKeyDown` is a no-op for these sub-modes.
- Icon uses a dynamic template (same pipeline as `session-info` / `telemetry-display`) rendering the live value with appropriate formatting (% for brake bias, integer slot for TC/ABS, %/index for mixture, etc.).

**Telemetry plumbing**

- For each View sub-mode implemented, add the relevant `dc*` field to `TelemetryData` in `packages/iracing-native/src/defines.ts`. Only wire what each sub-mode needs.

### Per-setting checklist

One PR per group is fine; check off as shipped.

- [ ] **setup-brakes** — View Brake Bias (`dcBrakeBias`), View Brake Bias Fine (`dcBrakeBiasFine`), View Peak Brake Bias (`dcPeakBrakeBias`), View Brake Misc (`dcBrakeMisc`), View Engine Braking (`dcEngineBraking`), View ABS Adjust (`dcABS`).
- [ ] **setup-traction** — View Traction Control (`dcTractionControl`), View TC Toggle (`dcTractionControlToggle`).
- [ ] **setup-fuel** — View Fuel Mixture (`dcFuelMixture`), View Fuel Cut Position (`dcFuelCutPosition`).
- [ ] **setup-engine** — View Engine Power (`dcEnginePower`), View Throttle Shape (`dcThrottleShape`), View Launch RPM (`dcLaunchRPM`).
- [ ] **setup-chassis** — View Anti-Roll Front (`dcAntiRollFront`), View Anti-Roll Rear (`dcAntiRollRear`), View Diff Entry / Middle / Exit (`dcDiffEntry` / `dcDiffMiddle` / `dcDiffExit`), View Power Steering (`dcPowerSteering`), View Weight Jacker Right (`dcWeightJackerRight`).
- [ ] **setup-hybrid** — View MGU-K Deploy Mode (`dcMGUKDeployMode`), View MGU-K Regen Gain (`dcMGUKRegenGain`), View MGU-K Deploy Fixed (`dcMGUKDeployFixed`).

### Affected packages

- `@iracedeck/iracing-native` — `TelemetryData` interface additions per setting wired.
- `@iracedeck/iracing-sdk` — confirm parser picks up the new scalar `dc*` fields automatically (it should).
- `@iracedeck/iracing-actions` — Setting dropdown additions, View sub-mode logic, dynamic icon template(s), telemetry subscription per View sub-mode.

### Documentation / website / skills

- For each implemented View sub-mode, document on the action page: the telemetry source, the displayed format, what value range to expect.
- `.claude/skills/iracedeck-actions/`, `.claude/skills/iracing-telemetry/` — reflect the newly wired telemetry vars.

### Tests

- Per View sub-mode: renders the value sent by the telemetry mock, hides Direction / dual-press controls in the PI, doesn't fire any binding on press.
- Format edge cases per setting (negative values, zero, out-of-range).

### Verification

- `pnpm install && pnpm build && pnpm test && pnpm lint:fix && pnpm format:fix`.
- In iRacing on the Stream Deck: configure each shipped View sub-mode on a key; adjust the setting in-game (via wheel rotary or the matching Adjust key); the View key updates within the next telemetry tick.

### Out of scope

- Dual-press behaviour — #540.
- View modes for non-setup actions (camera, push-to-pass already reflected via dedicated icons / states).
- Per-car value-range normalization (some sims expose 0–10 slot indices, not absolute %). Render what `dc*` provides; if a car-specific range is wanted later, file a follow-up.

### Additional context

- iRaceIT reference: https://iraceit.itxware.de/iraceit-the-streamdeck-plugin/incar/

Area: Action (existing)

### PR DESCRIPTION
**Pull request by niklam**
## Related Issue

Fixes #541

## What changed?

Each setup action now exposes a read-only **View …** entry per driver-control telemetry value. View entries render the live `dc*` value on the key via a shared dynamic icon template; key presses are no-ops. Pairs with the existing Increase/Decrease adjust modes — a typical layout is "View + adjust +/-" next to each other.

- **28 new View entries** across setup-aero (2), setup-brakes (6), setup-chassis (9), setup-engine (3), setup-fuel (2), setup-hybrid (3), setup-traction (4 = TC1–TC4).
- **Shared `setup-view` module** owns the View definitions: `ViewSettingId` union, `VIEW_DEFS` registry mapping each ID to a telemetry field, on-icon label, value formatter, and per-view font size override.
- **Dynamic icon template** (`icons/setup-view.svg`) inherits the action's category color scheme by reading defaults from a representative static icon passed in by each action — so a View key looks like the matching adjust keys (green for engine, blue for chassis, etc.) rather than a neutral blue.
- **Property Inspector dropdowns** restructured into `Adjust` / `Display value` optgroups, both alphabetised internally. Display items are prefixed "View".
- **TC slot naming:** the four TC slot entries (adjust + view) are now labelled `TC1`–`TC4` (was `TC Slot N`).
- **24 new `dc*` fields** added to `TelemetryData` (`dcBrakeBias`, `dcTractionControl[2-4]`, `dcMGUKDeployMode`, …). The iRacing-sdk parser auto-picks them up from shared memory, no translator wiring required.
- **Brake-bias formatter fix:** iRacing exposes `dcBrakeBias` already in percent units (e.g. `54`, not `0.54`). New `formatPercentRaw` helper appends `%` without multiplying.

## How to test

1. Configure a key as e.g. `View Brake Bias` on the Brake Setup action. Connect iRacing. Confirm the live brake bias value renders within the next telemetry tick.
2. Adjust brake bias in-car (via wheel rotary or the matching `Brake Bias +/-` key). Confirm the View key updates.
3. Press the View key. Confirm no SDK command / keypress fires.
4. Repeat one sample View per setup action (setup-aero View Front Wing, setup-engine View Engine Power, setup-traction View TC1, etc.).
5. Open the PI for any setup action: confirm the Setting dropdown shows `Adjust` and `Display value` optgroups with alphabetised entries.

## Checklist

- [x] Linked to an approved issue
- [x] New code has unit tests (`setup-view.test.ts` + per-action view sub-mode blocks)
- [x] Changes registered in all applicable plugins (Stream Deck, Mirabox) — both plugins rebuild without code edits
- [x] No unrelated changes included

## Summary by CodeRabbit

* **New Features**
  * Added read-only "View" sub-modes for car setup actions to show live telemetry on Stream Deck keys (brakes, chassis, aero, engine, fuel, hybrid, traction).

* **Improvements**
  * Reorganized setup menus into "Adjust" and "Display value" groups.
  * Direction control now hides automatically for view (read-only) modes.
  * Shortened traction labels from "TC Slot 1/2/3/4" to "TC1/TC2/TC3/TC4".
  * Total modes increased to 289 across setup actions.

* **Documentation**
  * Updated docs to list new view sub-modes and mode counts.

### CODE REVIEW

File: .claude/skills/iracedeck-actions/SKILL.md [hunk 1]
```diff
@@ -43,9 +43,9 @@ When asked about actions or controls:
 | View & Camera | 5 | 88 | FOV, replay, camera controls, broadcast tools |
 | Media | 1 | 7 | Video recording, screenshots, texture management |
 | Pit Service | 3 | 15 | Fuel, tires, compounds, tearoff, fast repair |
-| Car Setup | 7 | 44 | Brakes, chassis, aero, engine, fuel mix, hybrid/ERS, traction control |
+| Car Setup | 7 | 73 | Brakes, chassis, aero, engine, fuel mix, hybrid/ERS, traction control — adjustment modes plus read-only "View …" sub-modes that display live dc* telemetry on the key |
 | Communication | 2 | 34 | Chat, macros (15), whisper, toggle, reply, race admin commands |
-| **Total** | **31** | **260** | |
+| **Total** | **31** | **289** | |
```
[the remark this answers is not in the document]
Comment (niklam) [reply .claude/skills/iracedeck-actions/SKILL.md:48]: Fixed in a48b518b — inline summary now reads "31 actions with 289 modes" to match the table total.

File: .claude/skills/iracing-telemetry/SKILL.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ This is the **source of truth** for which variables are available in Mustache te
 | Category | Pattern | Count | Examples |
 |----------|---------|-------|----------|
 | Tire/Shock | `LF*`, `RF*`, `LR*`, `RR*`, `CF*`, `CR*` | 88 | `LFtempCM`, `RRwearL`, `LFshockDefl` |
-| In-Car Adjustments | `dc*` | 42 | `dcBrakeBias`, `dcTractionControl`, `dcDRSToggle` |
+| In-Car Adjustments | `dc*` | 42 | `dcBrakeBias`, `dcTractionControl`, `dcDRSToggle`. The setup-* actions' "View …" sub-modes (issue #541) consume these directly — see `packages/iracing-actions/src/shared/setup-view.ts` for the registry mapping each view-* setting to its dc* field. |
```
[the remark this answers is not in the document]
Comment (niklam) [reply .claude/skills/iracing-telemetry/SKILL.md]: Fixed in a48b518b — wrapped `setup-*`, `view-*`, and `dc*` in backticks so markdownlint no longer flags MD037.

File: packages/website/src/content/docs/docs/actions/car-setup/setup-chassis.md [hunk 3]
```diff
@@ -3,12 +3,28 @@ title: Setup Chassis
...
+| View Diff Exit | `dcDiffExit` | integer | car-dependent slot |
+| View Anti-Roll Front | `dcAntiRollFront` | integer | car-dependent slot |
+| View Anti-Roll Rear | `dcAntiRollRear` | integer | car-dependent slot |
+| View Power Steering | `dcPowerSteering` | integer | car-dependent slot |
+| View Weight Jacker Left | `dcWeightJackerLeft` | signed percent | ±a few % |
+| View Weight Jacker Right | `dcWeightJackerRight` | signed percent | ±a few % |
+
 ## Modes
 
 Select the mode from the **Setting** dropdown in the Property Inspector. Every chassis mode is directional — pick **Increase** or **Decrease** in the **Direction** setting to control what a button press does.
```
[the remark this answers is not in the document]
Comment (niklam) [reply packages/website/src/content/docs/docs/actions/car-setup/setup-chassis.md]: Fixed in a48b518b — Modes intro now reads "**Adjust** modes are directional… **View** modes are read-only and do not use Direction."

File: packages/website/src/content/docs/docs/actions/car-setup/setup-engine.md [hunk 4]
```diff
@@ -3,12 +3,22 @@ title: Setup Engine
...
+
+| View setting | Telemetry source | Format | Typical range |
+|---|---|---|---|
+| View Engine Power | `dcEnginePower` | integer | car-dependent slot |
+| View Throttle Shape | `dcThrottleShape` | integer | car-dependent slot |
+| View Launch RPM | `dcLaunchRPM` | integer | RPM |
+
 ## Modes
 
 Select the mode from the **Setting** dropdown in the Property Inspector. Every engine mode is directional — pick **Increase** or **Decrease** in the **Direction** setting to control what a button press does.
```
[the remark this answers is not in the document]
Comment (niklam) [reply packages/website/src/content/docs/docs/actions/car-setup/setup-engine.md]: Fixed in a48b518b — same Adjust/View clarification applied to setup-engine.md.
